menu.js

```diff
@@ -81,6 +82,17 @@ const mainMenuTemplate = (win) => {
 						config.setMenuOption("options.resumeOnStart", item.checked);
 					},
 				},
+				{
+					label: 'Starting page',
+					submenu: Object.keys(startingPages).map((name) => ({
```

Comment:
Is there to have no starting page once you have selected any? typically staying on the song page

Reply:
I'm not totally sure that I understand your question, but if you select the first option (`Home`) then it doesn't do anything since Home is the default starting page

Reply:
Unless I missed something, currently if "resume on last song" is checked, it will load the song URL (`https://music.youtube.com/watch?v=XXX`) - if Home is picked as starting page as well, it will redirect to `https://music.youtube.com/` (and so minimize the player), which is not the same behavior - having the option to unselect a previously starting page would resolve that!

Reply:
If home is selected, it won't do anything

https://github.com/th-ch/youtube-music/blob/f5758bfe939d60f7d9d562af17279a245e87a977/preload.js#L136-L138

Reply:
Isn't it `Home` with a [capital letter](https://github.com/th-ch/youtube-music/pull/1073/files#diff-c7db0c4eb7f7457e25664ef3b731ac3738f118ee769f698b938bd8bd5a409c4eR2)? 

But redirect to the home page feels like a good option, we should just allow unselecting the currently selected starting page for noop :)

Reply:
fixed by https://github.com/th-ch/youtube-music/pull/1073/commits/45fa963818f9209450d078b4ff75b7dfd9624210 (3 lines added)